integration-tests/oauth-mock-test-runner/src/main/groovy/com/okta/test/mock/wiremock/HttpMock.groovy

```diff
@@ -0,0 +1,356 @@
+
+        wrongScopeAccessTokenJwt =  Jwts.builder()
+                .setSubject("joe.coder@example.com")
+                .setAudience("api://default")
+                .claim("scp", ["profile", "openid"])
+                .claim("groups", ["Everyone", "Test-Group"])
+                .setIssuedAt(Date.from(now))
+                .setNotBefore(Date.from(now))
+                .setExpiration(Date.from(now.plus(1, ChronoUnit.HOURS)))
+                .setHeader(Jwts.jwsHeader()
+                    .setKeyId('TEST_PUB_KEY_ID'))
+                .signWith(SignatureAlgorithm.RS256, keyPair.privateKey)
+                .compact()
+
+        invalidSignatureAccessTokenJwt =  Jwts.builder()
+                .setSubject("joe.coder@example.com")
+                .setAudience("api://default")
+                .claim("scp", ["profile", "openid", "email"])
+                .claim("groups", ["Everyone", "Test-Group"])
+                .setIssuedAt(Date.from(now))
+                .setNotBefore(Date.from(now))
+                .setExpiration(Date.from(now.plus(1, ChronoUnit.HOURS)))
+                .setHeader(Jwts.jwsHeader()
+                .setKeyId('TEST_PUB_KEY_ID'))
+                .signWith(SignatureAlgorithm.RS256, invalidKeyPair.private)
+                .compact()
+                
+        wrongAudienceAccessTokenJwt =  Jwts.builder()
+                .setSubject("joe.coder@example.com")
+                .setAudience("api://wrong-default")
+                .claim("scp", ["profile", "openid", "email"])
+                .claim("groups", ["Everyone", "Test-Group"])
+                .setIssuedAt(Date.from(now))
+                .setNotBefore(Date.from(now))
+                .setExpiration(Date.from(now.plus(1, ChronoUnit.HOURS)))
+                .setHeader(Jwts.jwsHeader()
+                .setKeyId('TEST_PUB_KEY_ID'))
+                    .signWith(SignatureAlgorithm.RS256, keyPair.privateKey)
+                .compact()
+
+        idTokenjwt =  Jwts.builder()
+                .setSubject("00uid4BxXw6I6TV4m0g3")
+                .claim("name", "Joe Coder")
+                .claim("email", "joe.coder@example.com")
+                .claim("preferred_username", "jod.coder@example.com")
+                .setAudience("OOICU812")
+                .setIssuer("http://localhost:${getMockPort()}/oauth2/default")
+                .setIssuedAt(Date.from(now))
+                .setNotBefore(Date.from(now))
+                .setExpiration(Date.from(now.plus(1, ChronoUnit.HOURS)))
+                .setHeader(Jwts.jwsHeader()
+                    .setKeyId('TEST_PUB_KEY_ID'))
+                .signWith(SignatureAlgorithm.RS256, keyPair.privateKey)
+                .compact()
+
+        // client_secret_basic auth scheme is used by default (Basic client_id:client_secret) by the clients
+        authHeader = "Basic " + "OOICU812:VERY_SECRET".bytes.encodeBase64().toString()
+    }
+
+    void configureHttpMock(WireMockServer wireMockServer) {
```

Comment:
I think we will need to abstract this configuration out of this class.  We need to decouple the configuration from the server.  I also think we should only load the stubs needed for the scenario and not all of them.  This may help us both detect errors, run tests in parallel, and allow for specific scenario checks (i.e. did the discovery URL get called)